src/main.js

```diff
@@ -181,7 +181,15 @@ class PercyClient {
   getBuilds(project, filter) {
     filter = filter || {};
     let queryString = Object.keys(filter)
-      .map(key => 'filter[' + key + ']=' + filter[key])
+      .map(key => {
+        if (Array.isArray(filter[key])) {
+          // If filter value is an array, match Percy API's format expectations of:
+          //   filter[key][]=value1&filter[key][]=value2
+          return filter[key].map(array_value => 'filter[' + key + '][]=' + array_value).join('&');
```

Comment:
A little picky, but we could interpolate rather than concat:

```js
return filter[key].map(array_value => `filter[${key}][]=${array_value}`).join('&');
```

Reply:
Pickiness of the good kind.  Will update and merge.